node/core/approval-voting/src/tests.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
NOTE: Not all tests have been written and therefore some of the functions on the mock clock are considered dead_code. Pushing the PR up just to make sure the first steps are correct.

Reply:
is this going to be resolved in a follow up? what about old_tests?